CHANGELOG.md

```diff
@@ -617,6 +617,8 @@ Some of the highlights are:
  - `igraph_adjacency()` no longer accepts a negative number of edges in its adjacency matrix.
    When negative entries are found, an error is generated.
 
+ - The `igraph_eb_get_merges()` bridges parameter now starts the edge indices at 0, not 1.
```

Comment:
These are not edge indices. They are indices into the `result` vector, i.e. they represent the number of the step where the removal occurred.

Reply:
You're right, I wrote the changelog before reading your comments, and I forgot to fix it.